thebeetoken/beenest-web
# Don't show "started" bookings under Trips

## Description
Sending a user back to a `started` booking was implemented as a conversion-oriented convenience, but introduced some complexity: Allowing the user to complete that specific booking may violate business constraints (price quotes, reservations in particular) whereas sending the user to create a new booking leaves a `started` booking in our database and under the Trips tab.

Per standup, simplest solution is too remove `started` bookings from the Trips page.

## How to Test
1. Start to make a booking; do not pay!
2. Navigate to `/trips/current`
3. Verify that the booking you started does *not* appear

Which devices did you test on?

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
There is some related code that can be retired on the backend. We may also want to implement some different strategy to encourage completion of unfinished bookings.
